helm/values.yaml

```diff
@@ -18,11 +18,19 @@ container:
   readOnlyRootFilesystem: true
   allowPrivilegeEscalation: false
   apiGatewayUrl: http://mine-support-api-gateway.default
-  restartPolicy: Always
+  restartPolicy: Always  
+  
 ingress:
   serviceName: mine-support
   endPoint: mine-support
   server: defradev.com
 replicas: 1
 minReadySeconds: 5
 cookiePassword: passwordpasswordpasswordpasswordpassword
+
+livenessProbe:          
+  path: /health
```

Comment:
the convention in kubernetes is to call the end point `healthz`

Reply:
I found a few different examples of what to call the end point.  healtz is in the official k8s site so can change it to that.